src/js/main.js

```diff
@@ -402,7 +402,11 @@ function startProcess() {
                         break;
                     case 'cli':
                         // Add a little timeout to let MSP comands finish
-                        GUI.timeout_add('wait_for_msp_finished', () => TABS.cli.initialize(content_ready, GUI.nwGui), timeout);
+                        if (timeout) {
```

Comment:
@haslinghuis How we can know here if it need a timeout?

Reply:
Ok, it is aplied on pid tuning tab for now

Reply:
Timeout is declared on line 253 as 0. Assigned to 100 on line 258 if needed. So the timeout is now only applied when it's set.

Reply:
Ok, then maybe it should be if(timeout > 0)??

Reply:
Needed a bigger timeout 😕 

if (timeout) should be the same as if (timeout > 0)

Reply:
No, because timeout is always defined at 0,there is always timeout defined.

Reply:
Just tested, it takes the else clause when timeout === 0